### DISCUSSION TITLE
Looking for NFT generator web app

### DISCUSSION THREAD

**Discussion by marvin0o** [community/community#38389] [Other Feature Feedback, Questions, & Ideas]
Hello im looking for developer who can help me to build an own NFT Generator app with smart contract builder integrated.

**Comment by jfullstackdev** [comment]
how about this ? 

https://nft-generator.art/

never tried that, just tried searching it too

**Reply by marvin0o** [reply]
Thats an good example yes, but want my own version for on my website. Searching for developer with experience to build a nft generator.  Who can help me to build my own version.

**Reply by jfullstackdev** [reply]
maybe you'll be seeing developers here that are willing to work on that, just check each repos 

https://github.com/topics/nft-generator

you can raise an Issue, so you can communicate with them if they are willing

**Reply by marvin0o** [reply]
Hi thanks for the suggestion im now in contact with https://imintify.com for whitelabel NFT Generator.

**Comment by digitpk** [comment]
Use [Hashlips](https://github.com/HashLips) nft engine to generate NFTs

**Reply by marvin0o** [reply]
Hi yes already familiar with the Hashlip generation works good for myself. But want to offer a web based " no-code" generator for my clients.

**Comment by strong-728** [comment]
Hello, How are you?
How about this?
https://github.com/0x-blockchain/nft.art.generation
This is simple Node.js console NFT Generation App but I can implement it on the website with UI.
If you are interested in me, plesae let me know.
Thanks.
